shoop/front/views/product.py

```diff
@@ -57,8 +57,7 @@ def get_context_data(self, **kwargs):
         elif product.mode == ProductMode.PACKAGE_PARENT:
             children = (
                 product.get_all_package_children()
-                .language(language)
-                .order_by("translations__name")
+                .translated(get_language())
```

Comment:
You're right, that left out the ordering.

Also, didn't realize but this doesn't have any ordering for the basket contents, so I added that too. It's a little bit hacky since we have to do the queryset in the template. Alternatively, we could pass an "ordering" parameter to get_all_package_children, but not sure if that's a good idea since we wouldn't know whether or not it was a translated field, etc.